Sigma_specification.md

```diff
@@ -461,14 +445,14 @@ Examples ii:
 * `<Data Name="ServiceName">MpKsl4eaa0a76</Data>` will be `ServiceName`
 
 ### Special Field Values
-
+/// NEED REVIEW
+modifier `exists` ?
+///
```

Comment:
What is this section referring to?

Reply:
normally `not null` should be written as `|exists: true` now , and a null value `|exists: false` (I think).
But empty on the other hand is always `: ''`.